app/models/request.rb

```diff
@@ -24,7 +24,7 @@ class Request < ApplicationRecord
 
   acts_as_taggable_on :tags
 
-  after_create { Request.broadcast!(self) }
+  after_create :broadcast_request
```

Comment:
I didn't see a way to easily skip this callback without turning it into a method that can be skipped in the seeds file.